packages/core/src/feed.ts

```diff
@@ -0,0 +1,60 @@
+import { Subject, Observable, map } from 'rxjs'
+import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitID } from '@ceramicnetwork/streamid'
+import type { AnchorStatus } from '@ceramicnetwork/common'
+
+export type Document = {
+  id: CommitID
+  content: any
+  anchorStatus: AnchorStatus
+  metadata: StreamMetadata
+}
+
+class DocumentsSubject extends Observable<Document> {
```

Comment:
This could make it easier to pipe an observable of `StreamState` directly to this subject:
```suggestion
class DocumentsSubject extends Observable<Document> implements NextObserver<StreamState> {
```